template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Models/ManageViewModels/IndexViewModel.cs

```diff
@@ -1,21 +1,25 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
 using System.Linq;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Identity;
 
 namespace Company.WebApplication1.Models.ManageViewModels
 {
     public class IndexViewModel
     {
-        public bool HasPassword { get; set; }
+        public string Username { get; set; }
```

Comment:
I don't think it hurts to be explicit in the view model about what's going to be rendered in the view.